mac-android-emulator.sh

```diff
@@ -98,7 +98,7 @@ launch_emulator() {
         echo "API 16 is the oldest API level the KA app supports and API 25 is the newest API."
         api=$( get_input "API [23]: " "23" )
         echo "See https://developer.android.com/ndk/guides/abis.html for more information on Android ABIs."
-        abi=$( get_input "ABI [x86_64]: " "x86_64" )
+        abi=$( get_input "ABI [x86_64 if Intel, arm64-v8a if M1/M2]: " "arm64-v8a" )
```

Comment:
I noticed that this was out-of-date for M1/M2 users (which are going to be the primary folks installing this, going forward) so I've updated this.